Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -498,22 +499,24 @@ rankCard item aboveThreshold state =
         , textView
           [ width WRAP_CONTENT
           , height WRAP_CONTENT
-          , text $ (DS.take 12 item.goodName) <> (if DS.length item.goodName > 12 then "..." else "")
+          , text ((DS.take 12 item.goodName) <> (if DS.length item.goodName > 12 then "..." else "")) 
           , textSize FontSize.a_14
           , color if aboveThreshold || (item == currentDriverData && currentDriverData.rank > 0) then Color.white900 else Color.black800
+          , visibility $ if checkDriverWithZeroRides item aboveThreshold state then GONE else VISIBLE 
           ]
         ]
       , linearLayout
         [ width WRAP_CONTENT
         , height WRAP_CONTENT
-        , gravity RIGHT
+        , gravity $ if checkDriverWithZeroRides item aboveThreshold state then LEFT else RIGHT
+        , padding $ if checkDriverWithZeroRides item aboveThreshold state then (PaddingLeft 10) else (PaddingLeft 0)
```

Comment:
@Anuragini-2506 can you add this check inside PaddingLeft . 

PaddingLeft (if checkDriverWithZeroRides item aboveThreshold state then 10 else 0 )

Reply:
done